### ISSUE TITLE
Training time

### ISSUE THREAD

**Issue by saymamba** [xyl-507/FMTrack#5]
Excellent work! 
I have a quick question: how long did it take you to train it?  On my end, training one epoch on a 24GB 4090 takes over ten hours, which is crazy! 
By the way, I can only set the batch size to 4. I've scaled down the learning rate proportionally, but will this still significantly affect the training results?
